sagemaker-clarify/fairness_and_explainability/fairness_and_explainability.ipynb

```diff
@@ -19,22 +19,27 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    print("\nFeature values -- Label", training_data.iloc[selected_example])
</pre><p>Do we need this print? seems it's printing info about the plot itself.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
Yeah, I did think about it, I believe idea is to show both a tabular representation and plot based, hence kept it as it is.